### ISSUE TITLE
Deleting a View and Writing Changes doesn't persist/save

### ISSUE THREAD

**Issue by aneroid** [sqlitebrowser/sqlitebrowser#863]
### Details for the issue

TL;DR: After deleting a View, "Write Changes", Close DB and re-open DB, the view still exists.

I've got a database with a view which I'm trying to delete. I do the following steps:

1. Open DB
1. In "Database Structure", I select the View and then click "Delete View"
    - View is no longer present in the DB Structure tab
1. Click "Write Changes"
1. Close the DB
1. Open the DB
1. The view still exists

I has previously done all of these in one step: Delete view, Create new with the same name, Write changes, close and re-open. That's when I noticed the issue and tried re-doing it in single simple steps as above.

#### I'm opening this issue because:

- [ ] DB4S is crashing
- [x] DB4S has a bug
- [ ] DB4S needs a feature
- [ ] DB4S has another problem

#### I'm using DB4S on:

- [x] Windows: ( _version:_ _7_ )
- [ ] Linux: ( _distro:_ ___ )
- [ ] Mac OS: ( _version:_ ___ )
- [ ] Other: ___

#### I'm using DB4S version:

- [x] 3.9.1
- [ ] 3.9.0
- [ ] Other: ___

#### I have also:

- [ ] Tried out the nightly version: https://github.com/sqlitebrowser/sqlitebrowser#windows
- [x] Searched for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by aneroid** [comment]
Aaargh! Ignore. I had another Python process in the background which was still connected to it. I assume commits are meant to be ignored if another connection is still open, even if the other one hasn't made any changes, pending or committed.

Closing.
